docs-site/docs/user/export/working-with-exports.md

```diff
@@ -28,11 +28,33 @@ Prerequisites:
   * You must have at least the Analyst role to the project and/or group you will create the export from.
 
 To create a sample export:
-  1. Navigate to the [project](../project/projects/manage-projects#view-projects-you-have-access-to) or [group](../organization/groups/manage#view-groups) that contains the samples you would like to export.
-  2. From the left sidebar, select **Samples**
-  3. Click the checkbox of each sample you'd like to export
-  4. Click **Create Export**
-  5. A pop-up will appear asking if you'd like to give the export a name and whether you'd like to receive an e-mail notification when the export is ready to download. Neither of these are required to create the export.
+1. Navigate to the [project](../project/projects/manage-projects#view-projects-you-have-access-to) or [group](../organization/groups/manage#view-groups) that contains the samples you would like to export.
+2. From the left sidebar, select **Samples**.
+3. Click the checkbox of each sample you'd like to export.
+4. Click **Create Export**.
+5. In the dropdown, select **Sample Export**.
+6. A pop-up will appear asking the following:
+   - The file formats you'd like to include within the export. This allows you to filter out any file formats you do not wish to export. You are required to include at least one file format to create an export.
+   - If you'd like to give the export a name (not required).
+   - If you'd like to receive an e-mail notification when the export is ready to download (not required).
+
+## Create Linelist Export
+
+Prerequisites:
+  * A project must contain at least one sample, and that sample must contain metadata.
+  * You must have at least the Analyst role to the project and/or group you will create the export from.
+
+To create a sample export:
```

Comment:
This should read `To create a sample linelist export:`

Reply:
Thanks, fixed in ac1ecd63fabba137949d87474056d7d98bf6d518